src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -43,6 +45,14 @@ export class DBiamPersonenkontextRepo {
         return personenKontexte.map(mapEntityToAggregate);
     }
 
+    public async deleteByPerson(personId: PersonID): Promise<number> {
```

Comment:
Not needed if cascade is used. Otherwise permissions should be checked.

Reply:
Permissions to retrieve the kontexte is not in scope of this ticket I think? We implemented /personen endpoints under SPSH-605. Maybe we should do the same and secure the /personenkontexte.

Reply:
Ps: To check for permissions for kontexte there is a Ticket still in progress, so we can't do that here.